### ISSUE TITLE
Why are models always pessimistic about the future expectations?

### ISSUE THREAD

**Issue by Lufffya** [unit8co/darts#1672]
I trained an NHiTS model to predict future product sales, but most models predict declines and few predict increases. 
This is inconsistent with the facts. What is the reason?

**Comment by Luffffffy** [comment]
When I retrained the model on the latest dataset, the model seemed to be more normal and no longer pessimistic about future predictions(even if the situation did not significantly decline during the same period in history)

However, this seems to raise a question: How often do you update the model? For example, do you use the latest data to train the model every day, or do you use it every week or every month?

**Comment by madtoinou** [comment]
The performance and "optimisticness" of the models are heavily impacted by the dataset and the loss function used to train the model.

Decision tree-based models (`LGBMModel` and `XGBoost` for example) are bad at predicting values outside of the range seen during training, which could cause "pessimistic" predictions if the target increases in the forecasted period.

The frequency of model retraining ultimately depends on the nature of your dataset and the resources available : on one hand it's unrealistic to retrain a model every time a new values is recorded and on the other hand, waiting too long between retrain could lead to poor performance despite recent values availability. This trade-off is specific to each problem and it's ultimately the responsibility of the people developing and deploying the models to find the right balance.
